context/app/routes_cells.py

```diff
@@ -157,3 +191,27 @@ def all_indexed_uuids():
 
     except Exception as e:
         return {'message': str(e)}
+
+
+@blueprint.route('/cells/cells-in-dataset-clusters.json', methods=['POST'])
+def cells_in_dataset_clusters():
+    # For a single dataset we want to get the expression level of a given gene for all cells.
+    # (In our discussion, we started by thinking about the set of matching cells,
+    # and then showing expression levels for the two groups, but that’s not needed.)
+
+    uuid = request.args.get('uuid')
+    gene_name = request.args.get('gene_name')
+    min_gene_expression = request.args.get('min_gene_expression')
+    client = _get_client(current_app)
+
+    try:
+        cells = client.select_cells(where='dataset', has=[uuid])
+        cells_list = list(cells.get_list(values_included=gene_name))
+
+        # list() will call iterator behind the scenes.
```

Comment:
```suggestion
        cells_list = cells.get_list(values_included=gene_name)

```
`for` loop over `cells_list` should work as-is: Doesn't need to be an _actual_ `list`.